airgen/Cargo.toml

```diff
@@ -11,10 +11,14 @@ repository = { workspace = true }
 powdr-ast.workspace = true
 powdr-number.workspace = true
 powdr-analysis.workspace = true
-
 log = "0.4.17"
-
 itertools = "0.13"
+topo_sort = "0.4.0"
```

Comment:
The source repository was alive for three weeks, has a single contributor and uses unsafe features. Zero changes, pull requests or issue answers for three years.

Reply:
call_graph.rs contains a topo sort alg, but uses recursion.

Reply:
fair, I'll make the `call_graph` one generic and use it here as well

Reply:
done